dashboard/config/libraries/GameLabJr.interpreted.js

```diff
@@ -176,8 +176,12 @@ function whenPressedAndReleased(direction, pressedHandler, releasedHandler) {
   touchEvents.push({type: keyWentUp, event: releasedHandler, param: direction});
 }
 
+function whenSpriteClicked(sprite) {
+  return mouseWentDown("leftButton") && mouseIsOver(sprite);
+}
+
 function clickedOn(sprite, event) {
-  touchEvents.push({type: mousePressedOver, event: event, sprite: sprite});
+  touchEvents.push({type: whenSpriteClicked, event: event, sprite: sprite});
```

Comment:
Is the behavior that we are moving away from expected in any locations or can it be changed to a single 'fire' across the board?

Reply:
Sorry, I'm not sure I understand what you're asking?

Reply:
Clarification - I just want to make sure that in any place we are referencing clickedOn that it is acceptable to now have the 'whenSpriteClicked' behavior instead of the 'mousePressedOver'. Aka, is there any case where we want to keep the 'mousePressedOver'?

Reply:
Ah okay, sorry
clickedOn() is only called by the gamelab_clickedOn block, which has the text "when {} clicked", so this should be an okay change to make.